extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -196,6 +198,17 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * Returns true if this element is the child of an amp-ad element.  For use by
+   * network-specific implementations that don't want to allow themselves to be
+   * embedded directly into a page.
+   * @return {boolean}
+   */
+  isInAmpAdTag() {
```

Comment:
Probably not, but I think it's still an open question whether we're going to try to address #3976 in this PR or a later one.